src/command.rs

```diff
@@ -57,11 +59,12 @@ impl FromStr for Command {
         match s.to_lowercase().as_ref() {
             "init"     => Ok(Command::Init),
             "campaign" => Ok(Command::Campaign),
+            "campaignupdates" => Ok(Command::CampaignUpdate),
```

Comment:
Sorry, so what is the difference between a campaign, an update, and a campaign update? I thought I understood the first two, but I don't get what the third is. If it is a necessary step for actually launching a campaign, it might be a good idea to update the README to walk through that.

Reply:
an update in ota-cli is a multi target update. A campaign update is an update

Reply:
While we're at it, a doubt of my own: the "multi-target" stands for "multiple ECUs to receive the update", it's not related to the targets of Update, correct? That's my understanding but I'm still not sure I got that right.

Reply:
correct.